packages/create-block/docs/external-template.md

```diff
@@ -104,3 +104,41 @@ The following configurable variables are used with the template files. Template
 -   `style` (default: `'file:./style-index.css'`) – a frontend and editor style definition.
 -   `render` (no default) – a path to the PHP file used when rendering the block type on the server before presenting on the front end.
 -   `customBlockJSON` (no default) - allows definition of additional properties for the generated block.json file.
+
+### `variants`
+
+Variants are used to create variations for a template. Variants can override any `defaultValues` by provided their own.
+
+```js
+module.exports = {
+	defaultValues: {
+		slug: 'my-fantastic-block',
+		title: 'My fantastic block',
+		dashicon: 'palmtree',
+		version: '1.2.3',
+	},
+	variants: {
+		primary: {},
+		secondary: {
+			title: 'My fantastic block - secondary variant',
+		},
+	},
+};
+```
+
+Variants are accessed using the `--variant` flag i.e`--variant secondary`.
```

Comment:
Minor nitpick about no punctuation before or around `i.e...`, but I'd write this as:

```
Variants are accessed using the `--variant` flag, i.e`--variant secondary`.
```

Or:

```
Variants are accessed using the `--variant` flag (i.e`--variant secondary`).
```